presto-main/src/main/java/io/prestosql/operator/ExchangeClientConfig.java

```diff
@@ -83,13 +85,38 @@ public Duration getMaxErrorDuration()
         return maxErrorDuration;
     }
 
+    @Config("exchange.timeout-failure-detection")
+    public ExchangeClientConfig setTimeoutFailureDetection(boolean b)
+    {
+        this.timeoutFailureDetection = b;
+        return this;
+    }
+
+    public boolean getTimeoutFailureDetection()
```

Comment:
generally boolean configs should have names like **useTimeoutFailure** or **isTimeoutFailureDetectionEnabled**...

Reply:
fixed by changing configuration property name.